flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/JdbcConnectionOptions.java

```diff
@@ -90,6 +101,11 @@ public JdbcConnectionOptionsBuilder withDriverName(String driverName) {
             return this;
         }
 
+        public JdbcConnectionOptionsBuilder withExtendProperties(Properties extendProperties) {
```

Comment:
won't be better have the method to allow add one property 

```
private Properties properties = new Properties();
public JdbcConnectionOptionsBuilder withConfig(String config, String value) {
            this.properties.put(config, value);
            return this;
}
```

this way we could also start thinking on moving user, pass and driver to this, as they will be used always inside the properties..
simplifying some logic in [SimpleJdbcConnectionProvider.java](https://github.com/apache/flink-connector-jdbc/pull/115/files#diff-1a261021e67f437344c5b6b2b5da274ff6914c51d8b5163ef30f03e6ba3614c5)

```
public JdbcConnectionOptionsBuilder withUsername(String username) {
            this.properties.put("user", username);
            return this;
}
```